### ISSUE TITLE
Baked PDF - MathML Spike

### ISSUE THREAD

**Issue by edwoodward** [openstax/cnx#188]
### Description

**Some experiments to see if we can**
1. Control the display of Math using CSS on SVG files and HTML. We want consistent display on the web and in the PDF.
2. Speed up math conversion by using HTML from MathML Cloud
3. Lower the total size of the math files if we use HTML instead of SVG.

Math should be converted using MathML Cloud so we use MathJax for both web and PDF..

**Questions the Spike should answer**

1. Check the file size of the book PDFs for both HTML & SVG math. PrinceXml can apparently choke due to size/number of SVGs resulting PDF can apparently be 4x larger
2. Check how text selection works in the PDF. This is an accessibility issue
3. Check how highlighting operates in both output formats (PDF & online)
4. See if we can control the styling of elements in the SVG/HTML
5. What percentage of math is too wide (& would need to have MathJax split it into multiple lines and see if prince scales the SVG down. Does the same happen for HTML output?) See the baked-pdf repo for running the experiments (there are commandline flags to toggle the output mode)

Links:

- https://github.com/Connexions/baked-pdf/pull/21 Using mathjax-node rather than starting up a browser (makes it closer to using mathmlcloud)
- https://github.com/Connexions/baked-pdf/pull/28 Parallelizing in the browser

----------------
**Steps to Reproduce the bug or Details about Change:** _What, Who, Where and How?_

- **User Type:**
` [ ] ` Admin  ` [ ] ` Content/QA
` [ ] ` Student  ` [ ] ` Teacher
` [ ] ` N/A
- **Username:** 
- **URL:**  
- **Course number:**  
- **Course type:**  

- **Steps**

----------------
**Environment:**

- **Instance:**
` [ ] ` production ` [ ] ` qa ` [ ] ` staging
` [ ] ` dev ` [ ] ` content ` [ ] ` demo
- **OS:**
` [ ] ` Mac OSX ` [ ] ` Windows
` [ ] ` Android ` [ ] ` iOS
- **Browser:**
` [ ] ` Chrome ` [ ] ` Firefox ` [ ] ` Safari
` [ ] ` Opera ` [ ] ` IE ` [ ] ` Edge
- **Browser Version:**  

----------------
**Possible impact on a User:**

----------------
**Images or Screenshots** (_attach below_)

**Comment by karenc** [comment]
As I [commented](https://github.com/Connexions/cnx/issues/190#issuecomment-437921832) on #190, it is actually significantly slower for prince to generate a pdf from html math compared to svg math.  The pdf generated are also about the same size (27MB for Statistics).

**Comment by openstaxalina** [comment]
Adding release labels for tracking purposes only.

**Comment by brittweinstein** [comment]
@karenc can we close this issue? Is more detailed work represented elsewhere?

**Comment by karenc** [comment]
I think we're waiting on @helenemccarron and @lwtchu on the styling of math in html.
